s2spy/time.py

```diff
@@ -512,23 +512,35 @@ def __init__(
     def n_targets(self):
         return len(self.targets)
 
-    def add_interval(
+    def add_intervals(
         self,
         role: Literal["target", "precursor"],
         length: str,
         gap: str = "0d",
+        n: int = 1,
     ) -> None:
-        """Add an interval to the calendar. The interval can be a target or a precursor.
+        """Add one or more intervals to the calendar. The interval can be a target or a
+        precursor.
```

Comment:
[Pep257](https://peps.python.org/pep-0257/#multi-line-docstrings): Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description. The summary line may be used by automatic indexing tools; **it is important that it fits on one line and is separated from the rest of the docstring by a blank line**.

Reply:
Thanks Peter, I missed that one.